lib/process-services-cloud/src/lib/form/services/form-cloud.service.ts

```diff
@@ -44,6 +44,8 @@ export class FormCloudService extends BaseCloudService implements FormCloudServi
         super(adfHttpClient);
     }
 
+    fullScreenMode = new Subject<boolean>();
```

Comment:
I'm not sure that how we display the form (whether inset or fullscreen) should be in the form-cloud.service.ts file, where most of the methods are related to the business logic and not the representation.

Reply:
Form Rendering engine? Or what other class we have for dealing with the form display?

Reply:
Absolutely, I think it makes more sense to have this subject on the `FormRenderingService`. Thanks for the suggestion.